### PR TITLE
Remove deprecated endpoints & commands.

### PR DESCRIPTION
**Pull request by DFurnes**
#### What's this PR do?
~Spring~ Fall cleaning! This pull request removes Artisan commands that are no longer relevant & API endpoints that are no longer used by any other apps (mainly things we had built for the mobile app).

#### How should this be reviewed?
Tests should continue to pass!

#### Checklist
- [ ] Documentation added for changed endpoints.
- [ ] Tests added for new features/bug fixes.
- [ ] Post a message in #api if this includes something that causes a rebuild!

### CODE REVIEW

**Comment by DFurnes** [comment]
This also removes the Phoenix Ashes API requests to trigger Quicksilver registration emails, since those are now sent via Customer.io & the request was just a no-op.

**Comment by DFurnes** [comment]
I've also deleted the old `Token` model (which was used for the mobile app's "sessions"). I'll make a note to create a migration to drop this collection later on as well.

File: app/Console/Commands/CustomerIoMobileUserImport.php [hunk 1]
```diff
@@ -1,114 +0,0 @@
...
-
-namespace Northstar\Console\Commands;
-
-use Carbon\Carbon;
-use Northstar\Models\User;
-use Illuminate\Console\Command;
-use Illuminate\Support\Collection;
-use Northstar\Services\CustomerIo;
-
-class CustomerIoMobileUserImport extends Command
```
Comment (weerd) [inline app/Console/Commands/CustomerIoMobileUserImport.php]: You did well my friend. Goodbye for now.... ⚰️

File: app/Http/Controllers/AvatarController.php [hunk 2]
```diff
@@ -1,76 +0,0 @@
...
-use Northstar\Http\Transformers\UserTransformer;
-use Northstar\Auth\Scope;
-use Northstar\Services\AWS;
-use Northstar\Models\User;
-use Illuminate\Http\Request;
-use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
-use Symfony\Component\HttpKernel\Exception\UnauthorizedHttpException;
-use Gate;
-
-class AvatarController extends Controller
```
Comment (weerd) [inline app/Http/Controllers/AvatarController.php]: Why's the `AvatarController` out?

Comment (DFurnes) [reply app/Http/Controllers/AvatarController.php]: This was only used by the mobile app, and I think we'd want to handle this differently if we re-add this feature again in the future (for example, supporting web uploads).

Comment (weerd) [reply app/Http/Controllers/AvatarController.php]: Ok gotcha. Yeah, I think we'd eventually want something similar when supporting web uploads and customizing profiles 🎉

Comment (DFurnes) [reply app/Http/Controllers/AvatarController.php]: Agreed!

**Pull request merged**